test/unit/smart_answer_flows/benefit_cap_calculator_flow_test.rb

```diff
@@ -0,0 +1,27 @@
+require_relative "../../test_helper"
+require "smart_answer_flows/benefit-cap-calculator"
+
+module SmartAnswer
+  class BenefitCapCalculatorFlowTest < ActiveSupport::TestCase
+    context BenefitCapCalculatorFlow do
+      setup do
+        @benefits = {
+          first_benefit: :first_benefit_amount?,
+          second_benefit: :second_benefit_amount?,
+          third_benefit: :third_benefit_amount?,
+          fourth_benefit: :fourth_benefit_amount?
+        }
+
+        @selected_benefits = [:second_benefit, :fourth_benefit]
+      end
+      context "next question" do
+        should "go the the next selected benefit question" do
```

Comment:
If we stick with the implementation of `.next_benefit_amount_question` then I think it'd be good to add an assertion to this test to demonstrate that `@selected_benefits` is also modified.